pkg/domain/domain.go

```diff
@@ -1418,27 +1404,57 @@ func (do *Domain) Init(
 			return err
 		}
 	}
+	return nil
+}
+
+// Start starts the domain.
+func (do *Domain) Start() error {
```

Comment:
How about renaming it to `StartBackgroundLoops` and adding some comments that some SQL can still be executed before domain started.

Reply:
https://github.com/pingcap/tidb/pull/54913/commits/0006970132ba77981c24bd66b868d869e2599c4f

i think `Start` is clear enough